src/Modules/OrchardCore.Commerce/Views/Payment/Index.cshtml

```diff
@@ -1,16 +1,269 @@
+                                asp-items="Model.Regions"
+                                class="form-control content-preview-text address__region"
+                                required></select>
+                    </div>
+                </div>
+
+                @if (!string.IsNullOrEmpty(orderPart.ShippingAddress.UserAddressToSave))
+                {
+                    <input type="hidden" asp-for="OrderPart.ShippingAddress.UserAddressToSave" class="address__userAddressToSave" />
+                    <div class="mb-3">
+                        <div class="form-check">
+                            <input type="checkbox"
+                               class="form-check-input content-preview-select address__toBeSaved"
+                               data-val="true"
+                               data-val-required="The Value field is required."
+                               id="OrderPart_ShippingAddress_ToBeSaved"
+                               name="OrderPart.ShippingAddress.ToBeSaved"
+                               value="true">
+                            <input name="OrderPart.ShippingAddress.ToBeSaved" type="hidden" value="false">
+                            <label class="form-check-label" for="OrderPart_ShippingAddress_ToBeSaved">
+                                @T["Save this address"]
+                            </label>
+                        </div>
+                    </div>
+                }
+            </div>
+
+            <div class="mb-3">
+                <div class="form-check">
+                    <input type="checkbox"
+                           asp-for="BillingAndShippingAddressesMatch"
+                           class="form-check-input content-preview-select"
+                           data-val="true"
+                           data-val-required="The Value field is required.">
+                    <label asp-for="BillingAndShippingAddressesMatch" class="form-check-label">
+                        @T["Shipping Address and Billing Address are the same"]
+                    </label>
+                </div>
+            </div>
+
+            <div class="mb-3">
+                <label for="StripePaymentPart_PaymentIntentId_Text">PaymentIntentId</label>
+                <input class="form-control content-preview-text"
+                       dir="ltr"
+                       type="text"
+                       id="StripePaymentPart_PaymentIntentId_Text"
+                       name="StripePaymentPart.PaymentIntentId.Text">
+            </div>
+
+            <div class="mb-3">
+                <label for="StripePaymentPart_PaymentMethodId_Text">PaymentMethodId</label>
+                <input class="form-control content-preview-text"
+                       dir="ltr"
+                       type="text"
+                       id="StripePaymentPart_PaymentMethodId_Text"
+                       name="StripePaymentPart.PaymentMethodId.Text">
+            </div>
```

Comment:
Any Stripe-specific inputs should be in the `StripeCheckout` shape.
Also you don't need `<div>` and `<label>`, just an `<input hidden id="..." name="...">`.

Reply:
Now you don't need [these SCSS rules](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/a52468420ee40d12f633d03667b1ba84a5a4802e/src/Modules/OrchardCore.Commerce/Assets/Styles/credit-card-form.scss#L28-L36) either.